components/director/internal/domain/application/service.go

```diff
@@ -40,6 +40,7 @@ const (
 	managedKey           = "managed"
 	subaccountKey        = "Subaccount"
 	locationIDKey        = "LocationID"
+	xsappnameKey         = "xsappname"
```

Comment:
Better use the env var APP_SELF_REGISTER_DISTINGUISH_LABEL_KEY because the key has a very specific SAP name. Also there is an override in the management-plane-config repo for this value.
When you change this to the respective env variable, the unit test will probably fail because the local value for the env variable is `subscriptionProviderId` or something like that - not sure.